src/templates/index.html

```diff
@@ -344,6 +356,230 @@ <h2 class="info-title">Paper Info:</h2>
+                        <li>
+                            Press ctrl/cmd+ '+' or ctrl/cmd+ '-'
+                        </li>
+                        <li>
+                            Or change percentage from the input box
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    Select text from the paper
+                </li>
+                <li>
+                    Reach page faster by clicking on the button &nbsp;<label>&#9776;</label>&nbsp;
+                    and scroll through thumbnails of the pages
+                </li>
+            </ul>
+            </p>
+            <h2 id="cross-references-title">Cross References</h2>
+            <p> Hover the references and extract data from them.
+            <ul>
+                <li>
+                    Finding a reference and hover it
+                    <ul>
+                        <li>
+                            If the reference is about an image you will see it in a small popup and a button
+                        </li>
+                        <li>
+                            If the reference is about a text you will see an extraction of it and a button
+                        </li>
+                        <li>
+                            If the reference is about a page you will see just the button
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    If you click the button, the page of the reference (entire page) will be open and you will have a
+                    two page layout
+                    You can also select text from it.
+                </li>
+            </ul>
+            </p>
+            <h2 id="knowledge-graph-title">Knowledge Graph</h2>
+            <p> Create a knowledge graph showing the citation relations of all the papers in the "references section" of
+                the current paper.
+                This can help you to understand the relations between each work better.
+                Knowledge graph contains nodes representing papers that can be citing another paper or that can be cited
+                by another paper.
+                These relationships are visualized by the direction of an arrow on a link between two nodes.
+                Arrow pointing to node means that paper represented by that node is being cited in paper which is
+                represented by node at the other end of the link.
+                Click on the knowledge graph button in the navigation bar to generate the knowledge graph. Initially
+                displayed graph has a middle node representing the
+                paper that is currently being read and the nodes around it represent papers being cited by or citing
+                that paper.
+            <ul>
+                <li>
+                    To generate the knowledge graph click the button on the navbar <i class="material-icons">schema</i>
```

Comment:
navbar is not a word - it's "navigation bar"

Reply:
fixed